tidb-cloud/dbt.md

```diff
@@ -0,0 +1,223 @@
+---
+title: TiDB with dbt
+summary: Learn the use cases of dbt in TiDB.
+---
+
+# TiDB with dbt
+
+[Data build tool (dbt)](https://www.getdbt.com/) is a popular open-source data transformation tool that enables analytics engineers to transform data in their warehouses through SQL statements. Through the [dbt-tidb](https://github.com/pingcap/dbt-tidb) plug-in, analytics engineers working with TiDB can directly create forms and match data through SQL without having to think about the process of creating tables or views.
```

Comment:
Ask reader to read dbt offical doc first. I think a user can't understand this doc before he read the offical doc

Reply:
This is a tutorial about dbt-tidb. We have already given the dbt website here, so we need not advice readers to read dbt official doc first.